src/main/java/com/cruru/applicant/controller/dto/DashboardApplicantDto.java

```diff
@@ -11,7 +11,10 @@ public record DashboardApplicantDto(
         String name,
 
         @JsonProperty("created_at")
-        LocalDateTime createdAt
+        LocalDateTime createdAt,
+
+        @JsonProperty("is_rejected")
+        boolean isRejected
```

Comment:
DTO에 Rapper Type으로 통일했어서, 바꿔주시면 좋겠습니다.
```suggestion
        Boolean isRejected
```